### PR TITLE
Issue #222: Update the mapping for cache folder to the correct one which it is used on new PHP images from devOps

### LINKED ISSUE #222: Fix cache path for php images

### PR DESCRIPTION
**Pull request by aritomelo**
Fix #222 

This will fix the issue while running composer install.
The issue is due to the modification on new PHP images which set the cache folder in another path.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,12 +2,15 @@ version: '2'
 services:
   web:
     image: fpfis/httpd-php-dev:7.1
+    #image: fpfis/httpd-php-dev:5.6
     working_dir: /var/www/html
     ports:
-    - 8080:8080
+      - 8080:8080
```
Comment (drupol) [inline docker-compose.yml]: Unrelated change

File: docker-compose.yml [hunk 2]
```diff
@@ -7,7 +7,8 @@ services:
     - 8080:8080
     volumes:
     - .:/var/www/html # Non Mac users.
-    - ~/.composer:/var/www/.composer
+    - ~/.composer/cache:/var/www/.composer/cache
```
Comment (voidtek) [inline docker-compose.yml:10]: The `cache folder` should be a personal override of docker-compose file, because it's only for performance.

A PR was created on `fpfis/httpd-php-dev` to configure the cache folder for compose to `/cache`, it will be available soon.

After that you should create a docker-compose.cache.yml with 
```
version: '2'
services:
  web:
    volumes:
      - <local_cache_folder>:/cache
```

and use the command 
`docker-compose -f docker-compose.yml -f docker-compose.cache.yml up -d`

for more information and to have a workaround for now, please see https://webgate.ec.europa.eu/CITnet/jira/browse/OPENEUROPA-1515

**Review by voidtek — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (drupol) [reply docker-compose.yml:10]: Thanks @voidtek , this makes sense, going to close this PR.

**Comment by drupol** [comment]
After a discussion with @voidtek, I'm going to close this PR.

We cannot enforce users to use the cache. We should be shipping the uppermost basic docker configuration and let user customize it at will.

That said, once devOps will have their images fixed, we will add some documentation on how to enable caching because this can be very useful on Drone and locally.
